toolkit/tools/pkg/imagecustomizerlib/customizeos.go

```diff
@@ -29,6 +29,11 @@ func doOsCustomizations(buildDir string, baseConfigPath string, config *imagecus
 		return err
 	}
 
+	err = cleanUpImage(imageChroot)
```

Comment:
Thiis should be one of the last steps we take during customizations (before verity thou)

Reply:
Moved this step to before `enableOverlays()` and `enableVerityPartition()`